modules/node_modules/@ncigdc/routes/CartRoute.js

```diff
@@ -34,6 +35,15 @@ export default connect(state => (state.cart))(
       Container={CartPage}
       queryConfig={new CartRoute(props)}
       environment={Relay.Store}
+      render={({ error, props }) => {
```

Comment:
is it possible to have `<Loader />` be in the CartPage `<FileTable />` instead of the route so that the SummaryCards and CountCards don't need to reload on page change?